File: HackerRank Problem/Ruby/CompareTheTriplets.rb
#!/bin/ruby

require 'json'
require 'stringio'

# Complete the compareTriplets function below.
def compareTriplets(a, b)
    score_a = 0
    score_b = 0
    
    a.each_with_index do |value_a, index|
        value_b = b[index]
        
        case
        when value_a > value_b
            score_a += 1
        when value_a < value_b
            score_b += 1
        end
    end

    [score_a, score_b]
end

fptr = File.open(ENV['OUTPUT_PATH'], 'w')

a = gets.rstrip.split.map(&:to_i)

b = gets.rstrip.split.map(&:to_i)

result = compareTriplets a, b

fptr.write result.join " "
fptr.write "\n"

fptr.close()

File: HackerRank Problem/Python/CompareTheTriplets.py
#!/bin/python3

import math
import os
import random
import re
import sys

# Complete the compareTriplets function below.
def compareTriplets(a, b):
    arr = [0,0]# array to be returned as alice and bob's score respectively
    for i in range(3):
        if a[i]>b[i]:
            arr[0] = arr[0]+1
        if a[i]<b[i]:
            arr[1] = arr[1]+1
    return arr
            

if __name__ == '__main__':
    fptr = open(os.environ['OUTPUT_PATH'], 'w')
    a = list(map(int, input().rstrip().split()))
    b = list(map(int, input().rstrip().split()))
    result = compareTriplets(a, b)
    fptr.write(' '.join(map(str, result)))
    fptr.write('\n')
    fptr.close()
